core/domain/story_domain.py

```diff
@@ -665,8 +665,8 @@ def get_acquired_skill_ids_for_node_ids(self, node_ids):
             the story.
 
         Returns:
-            list(str)|None. The list of acquired skill ids for the
-                node ids or None, if no node is linked to them.
+            list(str). The list of acquired skill ids for the node ids.
+            If no aquired skill id is linked to them, the list will be empty.
```

Comment:
"acquired" is misspelled. Make sure to proofread please.

Reply:
Done.